.github/workflows/generate_cli_doc.yml

```diff
@@ -0,0 +1,67 @@
+name: Update API docs
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
```

Comment:
This will only update the API docs after each release; that seems appropriate. I'm curious if we want to also consider doing so after each merge to `develop`. What are your thoughts?

Reply:
I don't think generating docs with every merge to develop would be correct here. We only have one version of the docs for the release branch so I wouldnt want to confuse people by adding features which arent yet released into the docs.